src/components/Settings/LanguageSetting.tsx

```diff
@@ -19,32 +21,41 @@ type Props = {
 
 const LanguageSetting = ( { onChange }: Props ) => {
   const { t, i18n } = useTranslation();
-  const [availableLocales, setAvailableLocales] = useState<LocalesResponse>( [] );
-  const availableLocalesOptions = Object.fromEntries(
-    availableLocales.map( locale => [locale.locale, {
+  const [webLocales, setWebLocales] = useState<LocalesResponse>( [] );
+  const webLocalesOptions = Object.fromEntries(
+    webLocales?.map( locale => [locale.locale, {
       label: locale.language_in_locale,
       value: locale.locale
     }] )
   );
   const [localeSheetOpen, setLocaleSheetOpen] = useState( false );
 
+  const offlineLocalesOptions = Object.fromEntries(
```

Comment:
Now that I'm trying this, I'm not even sure this is even necessary. The only situation in which a user might need these locales is if they sign in, then go offline, then go to Settings... which is so rare I don't think we need to handle it. In all other cases they're going to see fresh or cached web locales. So I think we can remove this logic and just hide the language chooser if the user does takes that very unusual path to Settings.

Reply:
I was wondering about that. Will remove